targets/PandABox/const/PandABox-clks_impl.xdc

```diff
@@ -3,10 +3,24 @@
 # -------------------------------------------------------------------
 
 set_clock_groups -asynchronous -group clk_fpga_0
+set_clock_groups -asynchronous -group clk_fpga_1
 set_clock_groups -asynchronous -group EXTCLK_P
 set_clock_groups -asynchronous -group GTXCLK0_P
 set_clock_groups -asynchronous -group GTXCLK1_P
 
```

Comment:
This complex set of cross-clock exclusions is sufficiently obscure that I think it needs a comment to explain what's going on here!